bc/channel/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+from bc.core.models import AbstractDateTimeModel
+from bc.users.models import User
+
+from .utils.masto import masto_regex
+
+
+class Channel(AbstractDateTimeModel):
+    """
+    A "Channel" is a particular account on a service, which is used by
+    BCB to broadcast or to issue commands to BCB.
+    """
+
+    TWITTER = "t"
+    MASTODON = "m"
+    CHANNELS = (
+        (TWITTER, "Twitter"),
+        (MASTODON, "Mastodon"),
+    )
+    service = models.CharField(
+        help_text="Name of the service",
+        max_length=100,
+        choices=CHANNELS,
+    )
+    account = models.CharField(
+        help_text="Name of the account",
+        max_length=100,
+    )
+    account_id = models.CharField(
+        help_text=(
+            "Service's ID number, username, etc. for the account, "
+            "if applicable"
+        ),
+        max_length=100,
+    )
+    user = models.ManyToManyField(
+        User,
+        help_text="Users that belong to the channel",
+        related_name="channels",
+    )
+    enabled = models.BooleanField(
+        help_text="Disabled by default; must enable manually", default=False
+    )
+
+    def self_url(self):
+        if self.service == "twitter":
```

Comment:
```suggestion
        if self.service == self.TWITTER":
```

Reply:
Please grep the code for any other times we use `service` and swap it out for `Channel.TWITTER` or whatever, please.

Reply:
Got it!